simulate-travis.py

```diff
@@ -97,6 +97,12 @@
                 then we run independent mulled-build tests on
                 a successfully-built recipe. Use this argument to disable this
                 behavior''')
+ap.add_argument('--disable-local', action='store_true', help='''If you are on
```

Comment:
I really like this target syntax and gives a lot of control esp to osx folks with docker.